addons/project_issue/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,70 @@
+---Fields in module 'project_issue'---
+project_issue / project.issue            / categ_ids (many2many)         : DEL relation: project.category
+project_issue / project.issue            / tag_ids (many2many)           : NEW relation: project.tags
+# TODO DONE
+
+project_issue / project.issue            / kanban_state (selection)      : now required, default = normal
+# Nothing to do
+
+project_issue / project.issue            / message_follower_ids (many2many): not a function anymore
+project_issue / project.issue            / message_follower_ids (many2many): relation is now 'mail.followers' ('res.partner')
+project_issue / project.issue            / message_follower_ids (many2many): type is now 'one2many' ('many2many')
+# Concerns module Mail (has to be done in another migration script specifically for that module) ---> Nothing to do
+
+project_issue / project.issue            / progress (float)              : module is now 'project_issue_sheet' ('project_issue')
+# Nothing to do
+
+project_issue / project.issue            / section_id (many2one)         : relation is now 'crm.team' ('crm.case.section')
+project_issue / project.issue            / section_id (many2one)         : was renamed to team_id [nothing to to]
+# Nothing to do
+
+project_issue / project.issue            / version_id (many2one)         : DEL relation: project.issue.version
+# Feature removed ---> Nothing to do
```

Comment:
the feature is removed, so there is nothing to do -> ok.
but maybe the feature was interesting, and will be redevelopped in an OCA / custom modules. 
In that case, rename the field by None will rename the column with a prefix / suffix like '__openupgrade__V8_V9' and so the data will not be lost and could be reused by a migration script of the future OCA / custom modules.

regards.